packages/elm/index.js

```diff
@@ -14,9 +14,10 @@ module.exports = elm
  * @param {string}                  [options.cwd]           Custom location for your elm files.
  * @return {Function}
  */
-function elm (options, isProduction) {
-  options = options || {}
-  isProduction = isProduction || process.env.NODE_ENV === 'production'
+function elm (options = {}, isProduction) {
```

Comment:
But yeah... As I think about it. Since it's just for selecting the defaults: Maybe it would be best to just use `NODE_ENV === 'production'` for selecting the defaults, without parameter.